src/services/saveImagesToUSB.ts

```diff
@@ -0,0 +1,49 @@
+import { IService } from '../types';
+import { readdirSync, existsSync, mkdirSync, rename, statSync, readdir } from 'fs';
+import { USB_WRITE_PATH } from 'config';
+
+export const SaveImagesToUSB: IService = {
+    execute: async () => {
+        try {
+            //Check if USB is connected
+            const usbConnected = existsSync(USB_WRITE_PATH);
+            if (usbConnected) {
+                const directoryPath = USB_WRITE_PATH;
+                const files = readdir(directoryPath, (err, files) => {
+                    files.forEach((file) => {
+                        if (file.endsWith("jpg")) {
+
+                            const formattedDate = new Date().toISOString().split('T')[0];
+
+                            const directoryPath = `${USB_WRITE_PATH}/${formattedDate}`;
+
+                            //Check if directory exists for the day on which image was collected
+                            if (!existsSync(directoryPath)) {
+                                try {
+                                    mkdirSync(directoryPath, { recursive: true });
```

Comment:
Don't think `{ recursive: true }` is needed here, but not critical

Reply:
removed!

Reply:
@HariHive still there, maybe you didn't actually commit the latest?